company_website/templates/main_page.haml

```diff
@@ -29,9 +29,12 @@
 {% block extra_script %}
     {% include "main_page_partials/scripts.haml" %}
     {% javascript 'main' %}
-    %script{:async => "",
-        :defer => "",
-        :src => "https://maps.googleapis.com/maps/api/js?key={{ google_api_key }}&callback=initMap"}
+    :javascript
+      if(screen.width >= 720) {
+          document.write(
+              '<script src="https://maps.googleapis.com/maps/api/js?key={{ google_api_key }}&callback=initMap" async defer><'+
+              '/script>'
+          );
+      }
```

Comment:
This should be in `.js` file. Please move it there

Reply:
It can't because of `{{ google_api_key }}` variable :/

Reply:
Try to add `js` variable in haml file and then move `js` code to separate file

Reply:
Done